src/drop.js

```diff
@@ -22,5 +22,5 @@ var _xdrop = require('./internal/_xdrop');
  *      R.drop(3, [1,2,3,4,5,6,7]); //=> [4,5,6,7]
  */
 module.exports = _curry2(_dispatchable('drop', _xdrop, function drop(n, list) {
-  return n < list.length ? _slice(list, n) : [];
+  return n <= 0 ? list : _slice(list, n);
```

Comment:
I saw that discussion but figured the decision would end up on the other side of the fence. I'm against that decision as it can lead to some confusing side effects and breaches the API contract/expectation of not returning a new collection in some cases